src/thor/bidirectional_astar.cc

```diff
@@ -367,7 +358,7 @@ inline bool BidirectionalAStar::ExpandInner(baldr::GraphReader& graphreader,
 }
 
 template <const ExpansionType expansion_direction>
-bool BidirectionalAStar::Expand(baldr::GraphReader& graphreader,
+void BidirectionalAStar::Expand(baldr::GraphReader& graphreader,
```

Comment:
oversight I guess, it's not using the return value anywhere

Reply:
so i think i implemented deadend detection before we did another refactor to add expandinner. this probably should ahve been done when expandinner became a thing